ecosystem/python/sdk/examples/simple-nft-v2.py

```diff
@@ -0,0 +1,125 @@
+# Copyright © Aptos Foundation
```

Comment:
Let's merge the current nft2 and this one and expand on the tutorial just ever so slightly.